### PR TITLE
perforce: Add docs on locally testing sub-repo permissions

### PR DESCRIPTION
**Pull request by ryanslade**

### CODE REVIEW

File: content/departments/support/process/p4-enablement.md [hunk 1]
```diff
@@ -190,6 +190,64 @@ pass12349ers
...
+  ]
+```
+
+You can get the `P4.passwd` token above from running `p4 -u admin login -p -a` using the admin password from 1password. It expires every day so you’ll need to regenerate it occasionally.
+
+Updated your local SG user to include the verified e-mail address of one of the users configured on the dogfood instance, for example `alice@perforce.sgdev.org`. Alice is a user in the our dogfood instance with permissions set against the `//test-perms` instance.
+
+_NOTE_: Don’t modify the permissions in Perforce since they are used for integration testing.
+
+In order not to wait for permissions syncs, you can force one with this mutation:
```
Comment (indradhanush) [inline content/departments/support/process/p4-enablement.md]:
```suggestion
In order to not have to wait for a permissions sync to complete, you can force one with this mutation:
```

**Review by indradhanush — APPROVED** [review]
LGTM. Left some minor suggestions.

**Review by bobheadxi — COMMENTED** [review]
Nice, thanks!!

File: content/departments/support/process/p4-enablement.md [hunk 2]
```diff
@@ -190,6 +190,64 @@ pass12349ers
...
+
+```yaml
+"PERFORCE": [
+    {
+      "p4.port": "perforce.sgdev.org:1666",
+      "p4.user": "admin",
+      "p4.passwd": "REDACTED",
+      "depots": {
+        "//test-perms/"
+      ],
```
Comment (mollylogue) [inline content/departments/support/process/p4-enablement.md:211]: Should this be a `}` instead? I'm just trying to get this to work for me and getting json errors

Comment (mollylogue) [reply content/departments/support/process/p4-enablement.md:211]: Oh wait actually I think the open bracket is the one that needs to change to `[`

**Comment by mollylogue** [comment]
Do we need to set a feature flag locally to get the permissions to sync or should it do it automatically?

**Comment by ryanslade** [comment]
@mollylogue Thanks, not sure how I got those brackets mixed up since I copied from my actual config...

Fixed here:
https://github.com/sourcegraph/handbook/pull/2067

I also added details about how to actually enable sub-repo permissions and perforce. I assume you probably had those already.

I think that's everything I've done and I start my local dev environment using `sg start`

**Pull request merged**